merge master into live-based

luyajun0205 opened the pull request.

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

marstr committed:
Adding missed release notes.

Commit:
Copyedit release notes

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=530425602
https://github.com/Azure/azure-cli-extensions/commit/d095ff237a5488e5b4229a61396bb16b6db6879e

sptramer committed:
Merge pull request #1451 from marstr/fixup_release_notes

Adding missed release notes.

sptramer committed:
Merge pull request #1453 from MicrosoftDocs/az-ext-list-ci-530425602

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

v-susu-zz commented:
Docs Build status updates of commit _[a226d4b](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/pull/59/commits/a226d4bb96798ee4951377cddcabe57693b3f377)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-f87657b7)

<a id="f87657b7"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OPS-E2E-Prod/azure-docs-cli-python, working directory: W:\1fqb-s, target branch: live-based, target branch commit: 092466e03cc69b4f477d272d3c646c9c40499df1, source commit: a226d4bb96798ee4951377cddcabe57693b3f377.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C13%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C201905130702423932-59%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=O9y7geCH%2FRvEVa2W74y5IcErGKp8vklYadbhhlgAGgM%3D&st=2019-05-13T06%3A58%3A08Z&se=2019-06-13T07%3A03%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Merge branch 'live-based' into master

v-susu-zz commented:
Docs Build status updates of commit _[ce39e3c](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/pull/59/commits/ce39e3c1e947761c88a2c6f3d2c8494b61330dd3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[2018-03-01-hybrid/docs-ref-autogen/ams/asset.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2018-03-01-hybrid/docs-ref-autogen/ams/asset.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ams/asset?view=azure-cli-2018-03-01-hybrid&branch=pr-en-us-59) (azure-cli-2018-03-01-hybrid) |
[2019-03-01-hybrid/docs-ref-autogen/ams/asset.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2019-03-01-hybrid/docs-ref-autogen/ams/asset.yml) | :white_check_mark:Succeeded | |
[docs-ref-conceptual/azure-cli-extensions-list.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/azure-cli-extensions-list.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-extensions-list?view=azure-cli-latest&branch=pr-en-us-59) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-extensions-list?view=azure-cli-2017-03-09-profile&branch=pr-en-us-59) (azure-cli-2017-03-09-profile)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-extensions-list?view=azure-cli-2018-03-01-hybrid&branch=pr-en-us-59) (azure-cli-2018-03-01-hybrid) |
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-59) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-59) (azure-cli-2017-03-09-profile)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/release-notes-azure-cli?view=azure-cli-2018-03-01-hybrid&branch=pr-en-us-59) (azure-cli-2018-03-01-hybrid) |
[latest/docs-ref-autogen/ams/asset.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/latest/docs-ref-autogen/ams/asset.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ams/asset?view=azure-cli-latest&branch=pr-en-us-59) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C13%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C201905130703370969-59%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gWTsBE2aJThVWO0EHwMovnu7D0VhBHVKKCh34vVE7Mw%3D&st=2019-05-13T07%3A07%3A22Z&se=2019-06-13T07%3A12%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Add single-value example.

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

sptramer committed:
Merge pull request #1458 from sptramer/query/add-examples

Add single-value example.

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

yugangw-msft committed:
Updating release notes for 2.0.65

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=535444958
https://github.com/Azure/azure-cli-extensions/commit/e809b4c5efdad95dfe5e05218b5ac8349201a3b2

rloutlaw committed:
Merge pull request #1460 from yugangw-msft/cli-release-notes-2.0.65

Updating release notes for 2.0.65

rloutlaw committed:
Merge pull request #1461 from MicrosoftDocs/az-ext-list-ci-535444958

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

yugangw-msft committed:
azure-cli: fix markdown error in the releasing notes

rloutlaw committed:
Merge pull request #1463 from yugangw-msft/2.0.65-addon

azure-cli: fix markdown error in the releasing notes

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI update for deleted files

sptramer committed:
Merge pull request #1464 from MicrosoftDocs/master_delete

CI update for deleted files

Commit:
Copyedit 2.0.65 release notes

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=535916221
https://github.com/Azure/azure-cli-extensions/commit/eeaa85da97d5a24f8941fd53951ee886d99a34ee

sptramer committed:
Merge pull request #1467 from MicrosoftDocs/az-ext-list-ci-535916221

Update CLI extensions available doc

sptramer committed:
Merge pull request #1466 from sptramer/release-az-2.0.65

Copyedit 2.0.65 release notes

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)
